solidpy/Burn.py

```diff
@@ -216,29 +398,26 @@ def raw_simulation_data_export(self):
                     "Regressed Length",
                 ]
             )
-            for (
-                self.time,
-                self.chamber_pressure,
-                self.free_volume,
-                self.regressed_length,
-            ) in zip(
+            for (time, chamber_pressure, free_volume, regressed_length,) in zip(
                 self.solution.t,
                 self.solution.y[0],
                 self.solution.y[1],
                 self.solution.y[2],
             ):
-                self.evaluate_exit_pressure(self.chamber_pressure)
-                self.evaluate_exit_velocity()
-                self.evaluate_thrust(self.chamber_pressure)
+                exit_pressure = self.BurnSimulation.evaluate_exit_pressure(
+                    chamber_pressure
+                )
+                exit_velocity = self.BurnSimulation.evaluate_exit_velocity()
+                thrust = self.BurnSimulation.evaluate_thrust(chamber_pressure)
```

Comment:
Any reason for using this method for calculating thrust instead of the other one (`evaluate_thrust2`) which utilizes the thrust coefficient C_f?

It might be useful to have it as an intermediate simulation parameter so we can later apply corrections regarding nozzle inefficiencies (many of them can be found [here](rasaero.com/dloads/Departures%20from%20Ideal%20Performance.pdf)).

Also, have you checked if both methods end up returning the same results? If not, it might indicate something suspicious regarding the implementation of one (or maybe both) of them.

Reply:
These methods for thrust evaluation need revision, the first one was made in conformity with the equations described here: https://www.grc.nasa.gov/www/k-12/rocket/rktthsum.html

I checked some time ago and the methods did not output the same results.

Reply:
Marvelous, good catch! As expected, they should return the same thing, 
I see you've also reduced both methods into a single one, since they now output the same result.

This was worrying me quite a bit, glad to hear it's solved!